tracer/src/Datadog.Trace.Tools.dd_dotnet/CreatedumpCommand.cs

```diff
@@ -481,6 +516,23 @@ private unsafe void GenerateCrashReport(int pid, int? signal, int? crashThread)
             return;
         }
 
+        if (Environment.GetEnvironmentVariable("DD_INTERNAL_CRASHTRACKING_CRASH") == "1")
```

Comment:
Wow, you added a real variable that causes crashes into the production code. Bold move.

Reply:
:thuglife: